app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -157,6 +158,18 @@ def create_submission(saved_claim)
     end
 
     def log_failure(claim)
+      if Flipper.enabled?(:disability_526_track_saved_claim_error) && claim&.errors
+        monitor = DisabilityCompensation::Loggers::Monitor.new
+        in_progress_form = InProgressForm.form_for_user(FormProfiles::VA526ez::FORM_ID, @current_user) if @current_user
```

Comment:
I would surround this by a begin/rescue possibly? If there is a DB issue, or some error in `form_for_user` that raises here then the following still wouldnt get logged I dont think